app/views/community/stories/show.html.haml

```diff
@@ -31,22 +31,15 @@
   .md-container
     .bg-white.shadow-lg.rounded-16.py-32.px-40{ class: 'mt-[-64px]' }
       .flex.flex-row.mb-24
-        = graphical_icon "podcast-gradient", css_class: "h-[48px] w-[48px] mb-24"
+        = graphical_icon "podcast-gradient", css_class: "h-[48px] w-[48px] mr-24"
         .flex.flex-col
           %h2.text-h2.mb-8 More Stories from our community
           %p.text-p-large Listen, learn and be inspired by our community members.
       .grid.grid-cols-3.gap-16
         - @other_stories.each do |community_story|
           = link_to community_story_path(community_story), class: 'flex flex-col grid shadow-smZ1 p-16 bg-white rounded-8' do
-            .bg-borderLight.rounded-8.mb-12{ class: "w-[100%] pb-[56.25%]" }
+            = image_tag community_story.thumbnail_url, class: "bg-borderLight rounded-8 mb-12 w-[100%]", width: 243, height: 136
```

Comment:
This isn't actually on `main` but I can't rebase it out, which is confusing me here! 

@dem4ron @ErikSchierboom Any ideas?

Reply:
I had this wrong - I didn't realise it was duplicated. I've pushed this to main now. Thanks @dem4ron!